### PR TITLE
Multiclusterapp roles and system account changes

### PR DESCRIPTION
**Pull request by mrajashree**
Types PR: https://github.com/rancher/types/pull/718

### CODE REVIEW

File: pkg/controllers/management/multiclusterapp/handler_multiclusterapp.go [hunk 1]
```diff
@@ -163,37 +182,6 @@ func (mc *MCAppController) sync(key string, mcapp *v3.MultiClusterApp) (runtime.
...
-						RoleTemplateName: r,
-						ProjectName:      p.ProjectName,
-					})
-					if err != nil && !apierrors.IsAlreadyExists(err) {
-						return nil, err
-					}
-				} else {
-					return nil, err
-				}
-			}
```
Comment (prachidamle) [inline pkg/controllers/management/multiclusterapp/handler_multiclusterapp.go]: What is this change that we are deleting?

Comment (mrajashree) [reply pkg/controllers/management/multiclusterapp/handler_multiclusterapp.go]: Oh, this is the PRTB generation logic. On Friday before merging the original PRs we realized we needed CRTB generation too. So I added a same for loop for PRTB and CRTB, and forgot to delete this original for loop.
This is where it's handled now https://github.com/rancher/rancher/pull/17978/files#diff-2370d970aad60679ffe6572e455106ceR122

And what I'm deleting is just the duplicate code @prachidamle

**Comment by prachidamle** [comment]
LGTM

**Comment by mrajashree** [comment]
@prachidamle addressed those comments on types PR and added the part of finding user by label

**Comment by mrajashree** [comment]
@prachidamle addressed those comments on types PR and added the part of finding user by label as per your comment

**Comment by prachidamle** [comment]
thank you @mrajashree, LGTM

**Pull request merged**